app/models/order.rb

```diff
@@ -13,11 +13,18 @@ class Order < ApplicationRecord
   enum :status, { pending: 0, completed: 1, canceled: 2 }, default: :pending
 
   validates :shares, presence: true, numericality: { greater_than: 0 }
-  validate :sufficient_shares_for_sell, if: -> { transaction_type == "sell" }
-  validate :sufficient_funds_for_buy, if: -> { transaction_type == "buy" }
+
+  # only those orders that are pending can be updated
```

Comment:
Can you remove the comments please?  The `order_is_pending` validation is clear and this isn't necessary.